core/prototypes.js

```diff
@@ -2,150 +2,150 @@
-		while (k < len) {
-			currentElement = O[k];
-			if (searchElement === currentElement ||
-				(searchElement !== searchElement && currentElement !== currentElement)) {
-				return true;
-			}
-			k++;
-		}
-		return false;
-	};
+    Array.prototype.includes = function(searchElement) {
+        'use strict';
+        var O = Object(this);
+        var len = parseInt(O.length) || 0;
+        if (len === 0) {
+            return false;
+        }
+        var n = parseInt(arguments[1]) || 0;
+        var k;
+        if (n >= 0) {
+            k = n;
+        } else {
+            k = len + n;
+            if (k < 0) {
+                k = 0;
+            }
+        }
+        var currentElement;
+        while (k < len) {
+            currentElement = O[k];
+            if (searchElement === currentElement ||
+                (searchElement !== searchElement && currentElement !== currentElement)) {
+                return true;
+            }
+            k++;
+        }
+        return false;
+    };
 }
 
 //array fill
-if (typeof Array.prototype.fill != 'function') {
-	Array.prototype.fill = function(value) {
-
-		// Steps 1-2.
-		if (this == null) {
-			throw new TypeError('this is null or not defined');
-		}
-
-		var O = Object(this);
-
-		// Steps 3-5.
-		var len = O.length >>> 0;
-
-		// Steps 6-7.
-		var start = arguments[1];
-		var relativeStart = start >> 0;
-
-		// Step 8.
-		var k = relativeStart < 0 ?
-			Math.max(len + relativeStart, 0) :
-			Math.min(relativeStart, len);
-
-		// Steps 9-10.
-		var end = arguments[2];
-		var relativeEnd = end === undefined ?
-			len : end >> 0;
-
-		// Step 11.
-		var final = relativeEnd < 0 ?
-			Math.max(len + relativeEnd, 0) :
-			Math.min(relativeEnd, len);
-
-		// Step 12.
-		while (k < final) {
-			O[k] = value;
-			k++;
-		}
-
-		// Step 13.
-		return O;
-	};
+if (typeof Array.prototype.fill !== 'function') {
+    Array.prototype.fill = function(value) {
+
+        // Steps 1-2.
+        if (this == null) {
+            throw new TypeError('this is null or not defined');
+        }
+
+        var O = Object(this);
+
+        // Steps 3-5.
+        var len = O.length >>> 0;
+
+        // Steps 6-7.
+        var start = arguments[1];
+        var relativeStart = start >> 0;
+
+        // Step 8.
+        var k = relativeStart < 0 ?
+            Math.max(len + relativeStart, 0) :
+            Math.min(relativeStart, len);
+
+        // Steps 9-10.
+        var end = arguments[2];
+        var relativeEnd = end === undefined ?
+            len : end >> 0;
+
+        // Step 11.
+        var final = relativeEnd < 0 ?
+            Math.max(len + relativeEnd, 0) :
+            Math.min(relativeEnd, len);
+
+        // Step 12.
+        while (k < final) {
+            O[k] = value;
+            k++;
+        }
+
+        // Step 13.
+        return O;
+    };
 }
 
 // array.find
 if (!Array.prototype.find) {
   Array.prototype.find = function(predicate) {
     if (this === null) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 4.](https://www.codacy.com/app/theprojectshop/Kassy/file/2589168512/issues/source?bid=3282987&fileBranchId=3291254#l133)